python/TestHarness/TestHarness.py

```diff
@@ -1244,3 +1241,29 @@ def preRun(self):
 
     def getOptions(self):
         return self.options
+
+    # Helper tuple for storing information about a cluster
+    HPCCluster = namedtuple('HPCCluster', ['scheduler', 'apptainer_modules'])
+    # The modules that we want to load when running in a non-moduled
+    # container on INL HPC
+    inl_modules = ['use.moose', 'moose-dev-container-openmpi/5.0.5_0']
+    # Define INL HPC clusters
+    hpc_configs = {'sawtooth': HPCCluster(scheduler='pbs',
+                                          apptainer_modules=inl_modules),
+                   'bitterroot': HPCCluster(scheduler='slurm',
+                                            apptainer_modules=inl_modules)}
```

Comment:
Would be cool to see this as loading up a yaml file maybe. Since we now require `pyaml` to be available anyway. Would also enable other labs's HPC Clusters perhaps.

Reply:
Yep, I plan on extending this more in the future. We need to add more context for things like number of memory per core (so that we can add memory requirements to test specs), etc. That'll come in time